adr/0003-monitor-condition-and-data-selectors.md

```diff
@@ -0,0 +1,33 @@
+# 3. Monitor condition and data selectors
+
+Date: 2019-05-23
+
+Architecture issue: [#100](https://github.com/home-assistant/architecture/issues/100)
+
+## Status
+
+Accepted
+
+## Context
+
+Home Assistant components use mostly `CONF_MONITORED_CONDITIONS` or other config options that allow user to select which data the interface should send to backend. The user need known what the different data points means on setup of the component/platform. Later he can select the entity again, they he want see on UI.
+
+## Decision
+
+Home Assistant components/platforms should grap all available data and send this to backend. Available means data they are anyway sended from API to instance or if they need only spare resource to get all data.
+
+We should redouce selector logic mostly to end point if it make sense in context of interface. User should not read available documentations and API descrition to find out which data they want have.
```

Comment:
```suggestion
Integrations should only include selector logic if it make sense in the context of interface, like if it would require extra requests. User should not have to read the available documentation and API descriptions to find out which data they want have.
```